controller/workitem.go

```diff
@@ -246,11 +246,17 @@ func (c *WorkitemController) Reorder(ctx *app.ReorderWorkitemContext) error {
 				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "failed to reorder work item"))
 			}
 
+			// check if the workitems to reorder belongs to the space
+			_, err = appl.WorkItems().Load(ctx, ctx.SpaceID, wi.Number)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, "failed to reorder work item"))
```

Comment:
@hectorj2f is it better to write -> "unable to load workitem" because the error would not be because of "reorder", it would be because of failure in loading the workitem ??